### PR TITLE
Add more components to changelog template

### PR DESCRIPTION
**Pull request by pavolloffay**
This aligns components with labels on the repo

### CODE REVIEW

File: .chloggen/TEMPLATE.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
 change_type:
 
-# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opapm, github action)
```
Comment (swiatekm-sumo) [inline .chloggen/TEMPLATE.yaml:4]:
```suggestion
# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp bridge, github action)
```

Comment (swiatekm-sumo) [reply .chloggen/TEMPLATE.yaml:4]: If the label is just `opamp`, then I think it should be changed as well. WDYT @jaronoff97 ?

Comment (pavolloffay) [reply .chloggen/TEMPLATE.yaml:4]: I have changed the label to `opamp`

**Review by iblancasa — APPROVED** [review]

File: .chloggen/TEMPLATE.yaml (hunk 1 above)
Comment (swiatekm-sumo) [reply .chloggen/TEMPLATE.yaml:4]: In that case, you have a typo there.

Comment (pavolloffay) [reply .chloggen/TEMPLATE.yaml]: oh, fixed

**Review by swiatekm-sumo — APPROVED** [review]

**Pull request merged**